data/content-script.js

```diff
@@ -29,3 +29,6 @@ if (document.querySelector('#extra-verification-challenge')) {
     }
 
 }
+
+// put focus back on code field on submit in case of code error
+document.getElementById('verify_factor').addEventListener('click', focus_code_field);
```

Comment:
this should only happen when #extra-verification-challenge exists (code as stands results in a js error on the user/pass page).

it would probably be nice to select the text in the field too instead of just focusing the field.